src/Athena.Data/Migrations/src/20180215110432_Identity.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE IF NOT EXISTS users (
+  id UUID PRIMARY KEY NOT NULL REFERENCES students(id) ON DELETE CASCADE,
+  username TEXT NOT NULL,
+  normalized_username TEXT UNIQUE NOT NULL,
+  email TEXT NOT NULL,
+  normalized_email TEXT NOT NULL,
+  email_confirmed BOOLEAN NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS roles (
+  id UUID PRIMARY KEY NOT NULL,
+  name TEXT NOT NULL,
+  normalized_name TEXT UNIQUE NOT NULL
+);
+
+INSERT INTO roles (id, name, normalized_name)
+  VALUES ('f839e0f0-e29d-4c89-b406-51a52dbb08b5', 'Administrator', 'administrator')
+  ON CONFLICT (id) DO UPDATE SET
+    name = 'Administrator',
+    normalized_name = 'administrator';
```

Comment:
I need to double check but I think ASP.NET Core Identity may upper-case for normalized names. Might need to change this.